app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -0,0 +1,59 @@
+module Authentication
+  module AuthnJwt
+    module IdentityProviders
+      # Class for providing jwt identity from the decoded token from the field specified in a secret
+      class IdentityFromDecodedTokenProvider < IdentityProviderInterface
+        def initialize(authentication_parameters)
+          @resource_id = authentication_parameters.authenticator_resource_id
+          @decoded_token = authentication_parameters.decoded_token
+          @secret_fetcher = Conjur::FetchRequiredSecrets.new
+          @resource_class = ::Resource
+          @logger = Rails.logger
+        end
+
+        def provide_jwt_identity
```

Comment:
this should be `jwt_identity`. ruby methods that return objects should have the name of the object it returns

Reply:
Done in all places